### PR TITLE
upgrade nats server to 2.9.15

### PR DESCRIPTION
**Pull request by caleblloyd**
- upgrade nats to 2.9.14

### CODE REVIEW

**Review by samuelattwood — APPROVED** [review]
LGTM

File: docker-compose.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ services:
 
   nats-a:
     container_name: helix-alpha-nats-a
-    image: nats:2.9.9-alpine
+    image: nats:2.9.14-alpine
```
Comment (caleblloyd) [inline docker-compose.yaml:31]:
```suggestion
    image: nats:2.9.15-alpine
```

File: docker-compose.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ services:
 
   nats-b:
     container_name: helix-alpha-nats-b
-    image: nats:2.9.9-alpine
+    image: nats:2.9.14-alpine
```
Comment (caleblloyd) [inline docker-compose.yaml:45]:
```suggestion
    image: nats:2.9.15-alpine
```

File: docker-compose.yaml [hunk 3]
```diff
@@ -56,7 +56,7 @@ services:
 
   nats-c:
     container_name: helix-alpha-nats-c
-    image: nats:2.9.9-alpine
+    image: nats:2.9.14-alpine
```
Comment (caleblloyd) [inline docker-compose.yaml:59]:
```suggestion
    image: nats:2.9.15-alpine
```

**Review by caleblloyd — COMMENTED** [review]
Moving to 2.9.15

**Pull request merged**